pennylane/devices/torch_ops.py

```diff
@@ -117,6 +117,128 @@ def op_matrix(elements):
 I4_array[-4, -4] = 0
 I4 = op_matrix(I4_array)
 
+QNPI4_array = np.zeros((16, 16))
```

Comment:
Is there a simpler way to implement the torch version?

Reply:
I had written the matrices as it is for readability and looking at the previous implementation. I've now used `np.diag` (for diagonal matrices) and `np.fliplr` ( for off-diagonal matrix), thus making it a bit simpler.